bundles/core/org.eclipse.smarthome.core/src/main/java/org/eclipse/smarthome/core/items/GenericItem.java

```diff
@@ -412,6 +414,53 @@ public StateDescription getStateDescription(Locale locale) {
         return null;
     }
 
+    @Override
+    public StateDescription getMixedStateDescription() {
+        return getMixedStateDescription(null);
```

Comment:
The JavaDoc tells the default locale is used. You pass in `null` here instead. Please also review the `getStateDescription` method which behaves analogous. Alternatively the JavaDoc in `Item` should be changed.